justfile

```diff
@@ -1,5 +1,7 @@
 set dotenv-load := false
 
+IS_CI := env_var_or_default("CI", "")
```

Comment:
[TIL here as well!](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) This is great